3/T4/index.js

```diff
@@ -0,0 +1,59 @@
+const min = document.getElementById('min');
+const max = document.getElementById('max');
+const buttonFindFriendlyNumbers = document.getElementById('find');
+
+buttonFindFriendlyNumbers.addEventListener('click',() =>{
+    if(+min.value < +max.value){
```

Comment:
I entered `100.1` and `300` and received empty array

Reply:
I entered `220` and `284` and received nothing

Reply:
done